build.sbt

```diff
@@ -57,17 +57,21 @@ scalacOptions ++= Seq("-feature")
 
 libraryDependencies ++= Seq(
   jdbc,
-  cache,
+  ehcache,
   ws,
-  "com.gu" %% "membership-common" % "0.608",
-  "com.gu.play-googleauth" %% "play-v27" % "1.0.7",
+  "com.gu" %% "membership-common" % "0.610",
+  "com.gu.play-googleauth" %% "play-v28" % "2.2.2",
   "com.softwaremill.macwire" %% "macros" % "2.5.0" % "provided",
   "com.softwaremill.macwire" %% "util" % "2.5.0",
   "com.softwaremill.macwire" %% "proxy" % "2.5.0",
   "ch.qos.logback" % "logback-classic" % "1.2.3",
   "org.scalatestplus.play" %% "scalatestplus-play" % "5.1.0" % Test
 )
 
+// Force jackson-databind version to avoid incompatibility with Play
+val jacksonDatabindVersion = "2.11.4"
+dependencyOverrides += "com.fasterxml.jackson.core" % "jackson-databind" % jacksonDatabindVersion
```

Comment:
this is annoying because snyk actually flags this as a medium severity, but I get errors if I force it to a newer version. I think we have to wait for Play to update

Reply:
yes I'm not sure if you know but you have to update to the latest patch version in the same minor version.  Going to the latest causes runtime errors due to it checking explicitly.  Fortunately jackson seem to maintain at least 3 minor versions, but you have to read the snyk affected versions expression carefully to work it out.